transform/mattermost-analytics/tests/staging/stripe/test_subscription__onprem_has_license_data.sql

```diff
@@ -27,14 +17,16 @@ where
     and p.name <> 'Premier Support'
     -- Skip incomplete subscriptions
     and s.status <> 'incomplete_expired'
+    -- Ignoring subscriptions that come from admin portal/sales
+    and s.renewal_type is null
```

Comment:
Is "subscriptions that come from admin portal/sales" == "renewal_type is null" ?

Reply:
> Is "subscriptions that come from admin portal/sales" == "renewal_type is null" or "renewal_type <> 'sales-only'"? Open question if/why "admin portal == sales_only".

According to `Self-serve` document:

> When a renewal type is sales-only, that subscription cannot be renewed self serve, and it doesn’t necessarily fit into the criteria of this document.

The values are either `sales-only` or `null`. It's expressed  this way here in order to explicit about this expectation.

Reply:
Makes sense, could be useful to add a link to the documentation/Confluence page for future reference.

Reply:
Adding Nick's comment here - The easiest way to determine if something was a sales-serve subscription vs not is by looking for a SUBSCRIPTIONS.METADATA key cws-purchase-source and value admin. Any subscription with that value was done so sales-serve.

Reply:
Thanks, added it to the Confluence page. Still not completely clear from his comment if ALL the sales-serve subscriptions follow this rule.